.bingo/faillint.mod

```diff
@@ -2,4 +2,4 @@ module _ // Auto generated by https://github.com/bwplotka/bingo. DO NOT EDIT
 
 go 1.19
```

Comment:
Why did we update the go version on `.bingo/golangci-lint.mod` but not in here?

Reply:
I believe this is what bingo is doing internally. golangci-lint requires go 1.22 but faillint not?!?